source/frontend/src/features/mapSideBar/tabs/takes/update/models.ts

```diff
@@ -28,6 +28,10 @@ export const TakesYupSchema = Yup.object().shape({
           .required('Required when flag is true')
           .moreThan(0, 'Must be greater than 0 when flag is true'),
       }),
+      ltcEndDt: Yup.string().when('isLicenseToConstruct', {
```

Comment:
this is the error that prints when ltc end date is not specified: Must be greater than 0 when flag is true